docker-compose.yml

```diff
@@ -26,6 +26,7 @@ services:
         - 8081:8081
```

Comment:
You'll also need to an environment variable here for the connection string, so we can override it and point to `db` instead of `localhost`